report_xlsx/models/ir_report.py

```diff
@@ -26,7 +26,7 @@ def _render_xlsx(self, report_ref, docids, data):
             .sudo(False)
             .create_xlsx_report(docids, data)  # noqa
         )
-        if ret and isinstance(ret, (tuple, list)):  # data, "xlsx"
+        if ret and isinstance(ret, (tuple | list)):  # data, "xlsx"
```

Comment:
Why did we use bitwise OR?

Reply:
With python 3.10, we should use `|`, you can find more info here: https://peps.python.org/pep-0604/